logs/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
Indeed! Spiffy, right?  Found that here, I believe: https://stackoverflow.com/a/932982